gr-dtv/lib/dvb/dvb_ldpc_bb_impl.h

```diff
@@ -35,7 +35,8 @@ class dvb_ldpc_bb_impl : public dvb_ldpc_bb
     int ldpc_lut_index[FRAME_SIZE_NORMAL];
     void ldpc_lookup_generate(void);
 
-    uint16_t** ldpc_lut;
+    std::vector<uint16_t*> ldpc_lut; // Pointers into ldpc_lut_data.
```

Comment:
Thanks for adding that comment! It's really helpful in this context. It avoids the "uh why do we still have raw pointers" question.